# Biograpgy
I am a temporary **research and teaching assistant in [MIDI](https://www.etis-lab.fr/la-recherche/equipe-midi/) team (Etis,Cergy Pontoise, France)**. 
My work focuses on **methods and data structures** in **sequence bioinformatics**, with applications to **pangenome** in particular.


After a bachelor's degree in computer science and a master's degree in Science and Digital for Health,
Bioinformatics, Knowledge, Data at the University of Montpellier, I did my PhD within
of the GAIA doctoral school (584), between [Cirad](https://www.cirad.fr/en/) and [LIRMM](https://www.lirmm.fr/). My thesis focuses on the development of a
_k_-mers indexing data structure intended to support the analysis of hundreds or even thousands of similar genomes. 
This general purpose data structure is intended to store information about many close genomes, but can also be adapted to store information on collections
genomes derived from sequencing, upstream or downstream of classification into families of related species.
([detailed cv](cv.md)).

# Ongoing work
- 2021 [Toward optimal fingerprint indexing for large scale genomics](https://www.biorxiv.org/content/biorxiv/early/2021/11/05/2021.11.04.467355.full.pdf), C.Agret, B.Cazaux, A.Limasset 
- 2020 [Redoak: a reference-free and alignment-free structure for indexing a collection of similar genomes](https://www.biorxiv.org/content/early/2020/12/21/2020.12.19.423583),C.Agret, A.Chateau, G.Droc, G.Sarah, A.Mancheron,  M.Ruiz.


## GenomeHarvest project:
- 2019:  When RedOakâ„¢ draws trees, https://www.genomeharvest.fr/results/seminars
- 2019: Poster: Resources to manage Rice Big Genomics Data
- 2018: Presentation of a tool under development allowing the indexing of a large collection of
unassembled genomes, https://www.genomeharvest.fr/results/seminars
- 2017: [Pangenome Indexing Structure applied to rice](https://www.genomeharvest.fr/results/seminars)


## SeqBIM: working group common to the GDR Molecular Bioinformatics (BIM) at the GDR Informatique
CNRS Mathematics (IM):
- 2018: SeqBio: Development of indexing compressed structure for analyzing a large collection of simi-
lar genomes: application to rice, URL: http://bioinfo.univ-rouen.fr/seqbio2018/proceedings.
pdf
- 2017: SeqBio: Compressed structure for analyzing a collection of similar genomes: application to rice,
URL: https://bioinfo.cristal.univ-lille.fr/seqbio2017/seqbio_abstracts.pdf


## JOBIM: Biology, Computer Science and Mathematics Open Days, the conference of the French Society
of Bioinformatics:
- 2017: Poster: Development of indexing compressed structure for analyzing a collection of similar
genomes: application to rice.
- 2015: Poster: Use of Illumina sequencing for accessing viral genomes Cocoa swollen shoot virus
(CSSV).


# Journal Publications
- 2018 [Next generation sequencing elucidates cocoa badna virus diversity and reveals the existence of more than ten viral species.](https://linkinghub.elsevier.com/retrieve/pii/S0168170217306743), E.Muller, S.Ravel, C.Agret, F.Abrokwah, H.Dzahini-Obiatey, I.Galyuon, K.Kouakou, E.C.Jeyaseelan,
J.Allainguillaume, A.Wetten.


# Communications
## Invited talks
- 2020 TODO
- 2017 Machine learning science popularization of supervised and unsupervised learning, MAB team seminar, LIRMM Montpellier (France)
- 2017 TODO, Cirad, Montpellier (France)


## Conferences
- 2020 [Todo _k_-mer index](todo.todo)


## Other
- 2020 Co-organization of the Open Days of Biology, Computer Science and Mathematics Congress of 30
June to July 3, 2020. I helped with logistics management and I was a support in the evaluation of productions
scientists. https://jobim2020.sciencesconf.org/
- 2018 Co-creator and co-organizer and Agapiades. This is the first symposium organized by the doctoral
rants from Agap for the personnel of the various research centers of Cirad, INRA, IRD,
CNRS and SupAgro. I took care of the general organization, the communication and proofreading part
scientific productions.[Agapiades](https://agapiades.sciencesconf.org/)
- 2016 Co-organizer of Montpellier Omics days 2016 (MOD2016). MODs are a scientific event.
tific organized by master's students for biologists and bioinformatics scientists. The days-
born are generally divided into two days, a day of colloquium and one or two days of training. In
- 2016 Co-organized the MODs, I actively chose the subject and the participants, with Y.COGNE and A.VERDIER
we prepared and gave training on <<From Linux to Differential Gene Analysis (DESeq2)>>. [MOD2016](https://mod2016.wordpress.com/)
- 2014 Co-creator, then president of the AsTICo association of the Master Sciences and Digital for Health.
The purpose of this association is to create a directory of alumni to allow inter-generational communication.
nationally, facilitate the organization of scientific events such as MODs, and facilitate the exchange of
course support between teachers and students. (AsTICo)[http://www.astico.fr/]


# Supervision
- 2020-2021 : A.Cuhadar (M2 internship), subject: Maize pangenomics study.
- 2019-2020 : N.Mijovic (M2 internship),subject: Comparison of a large collection of rice genomes using Formal Concept Analysis (FCA) and Data mining.
- 2017-2018 : L.Djardin Bretones (M1 internship),subject: Study of succinct structures for the compression of binary vectors.


# Teaching
<table>
  <tr>
    <th>Year</th><th>Level</th><th>Topic</th>
  </tr>
   <tr>
    <td>TODO</td>
  </tr>
   <tr>
    <td>2014/2015</td><td>TODO</td>
  </tr>
</table>
